tests/integration_tests/lightgbm_tuner_tests/test_optimize.py

```diff
@@ -1,8 +1,13 @@
 import contextlib
-
 import mock
```

Comment:
Ops, a tiny.
```suggestion

import mock
```

Reply:
Thanks! Applied it in https://github.com/optuna/optuna/pull/1032/commits/1ee06dc02d118860795f9d9eb57d2539daa6f88b.